services/spar/src/Spar/API.hs

```diff
@@ -1,6 +1,7 @@
-{-# LANGUAGE GeneralizedNewtypeDeriving #-}
 {-# LANGUAGE RecordWildCards #-}
 {-# OPTIONS_GHC -fplugin=Polysemy.Plugin #-}
+{-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
+{-# HLINT ignore "Use $>" #-}
```

Comment:
For this particular case, it makes (IMO) the code harder to read, with added parens and less uniform formatting.

```haskell
authreqPrecheck msucc merr idpid =
  ( validateAuthreqParams msucc merr
      *> getIdPConfig idpid
  )
    $> NoContent
```